src/app/layout/notification-bar/notification-bar.service.ts

```diff
@@ -94,7 +86,32 @@ export class NotificationBarService {
         this.displayedNotifications.next(this.notifications.slice(0, maxDisplayLength));
     }
 
-    public showNotification(notification: Notification) {
+    private createNotification(notificationContent: NotificationContent, options?: NotificationOptions) {
+
+        options = options || ({} as NotificationOptions);
```

Comment:
Don't override the input, create another one or default in signature (`options?: NotificationOptions = {}`).